client/components/olark-chat/index.jsx

```diff
@@ -11,6 +11,19 @@ class OlarkChat extends Component {
 			;(function(o,l,a,r,k,y){if(o.olark)return; r="script";y=l.createElement(r);r=l.getElementsByTagName(r)[0]; y.async=1;y.src="//"+a;r.parentNode.insertBefore(y,r); y=o.olark=function(){k.s.push(arguments);k.t.push(+new Date)}; y.extend=function(i,j){y("extend",i,j)}; y.identify=function(i){y("identify",k.i=i)}; y.configure=function(i,j){y("configure",i,j);k.c[i]=j}; k=y._={s:[],t:[+new Date],c:{},l:a}; })(window,document,"static.olark.com/jsclient/loader.js");
 			olark.identify('${ identity }');
 		`;
+
+		if ( shouldDisablePreChatSurvey ) {
+			script.innerHTML += `
+				olark.configure("features.prechat_survey", false);
+			`;
+		}
+
+		if ( systemsGroupId ) {
+			script.innerHTML += `
+				olark.configure('system.group', '${ systemsGroupId }'); 
+			`;
```

Comment:
Note - this helps route the incoming user to the pre-sales or signup user group defined in https://www.olark.com/team/groups.